### DISCUSSION TITLE
[Error] Umi-OCR is already running, HTTP cross process transmission instruction failed.

### DISCUSSION THREAD

**Discussion by Mike001-wq** [hiroi-sora/Umi-OCR#720] [Q&A]
after my input: "./umi-ocr.sh   --path   "example.png"    --output  result.txt" on Linux, an arror occurs:
Use the Python embeddable environment.
pwd: Umi-OCR_Project/Umi-OCR
Use headless mode.
_XSERVTransSocketUNIXCreateListener: ...SocketCreateListener() failed
_XSERVTransMakeAllCOTSServerListeners: server already running
(EE) 
Fatal server error:
(EE) Cannot establish any listening sockets - Make sure an X server isn't already running(EE) 
Umi-OCR 已在运行，HTTP跨进程传输指令失败。
[Error] Umi-OCR is already running, HTTP cross process transmission instruction failed.
http://127.0.0.1:1224/argv
error: <urlopen error [Errno 111] Connection refused>

Success output to file: Umi-OCR_Project/Umi-OCR/result.txt

**Comment by qwedc001** [comment]
请提供一下你的系统环境
